### DISCUSSION TITLE
开发一个安卓bilibili脚本，注入apk程序实现相应的功能可能吗

### DISCUSSION THREAD

**Discussion by passerbyo** [the1812/Bilibili-Evolved#3474] [功能建议 / Ideas]
开发一个安卓bilibili脚本，注入apk程序实现相应的功能可能吗

**Comment by Geeyun-JY3** [comment]
我建议还是死了让 Bilibili Evolved 搞什么手机版或移动端相关功能这条心：#102、#3306

而且目前 Android 端不有的是模块和修改版安装包吗？
